modules/com_vtiger_workflow/language/en_us.lang.php

```diff
@@ -224,6 +224,12 @@
 'HELP_GenerateImageCode_ValueToEncode' => 'Field name where the value to encode will be read from',
 'HELP_GenerateImageCode_EncodingType' => 'Encoding type to be used for the image',
 'HELP_GenerateImageCode_SaveTo' => 'Image field where the image will be saved',
+// watermark workflow context variable
+'HELP_WaterMark_Image_value' => 'Watermark image url',
+'HELP_Image_Field_Name' => 'The field of the image that gets watermarked',
+'HELP_WaterMark_Size' => 'The size of the watermark image in percentage',
+'HELP_WaterMark_Position' => 'the position of the watermark in image',
+'HELP_Expression_Type' => 'expresstion type',
```

Comment:
we have to add here the name, label, and class of the workflow task, they are used in some parts of the workflow system

```php
'Watermark' => 'Watermark',
'CBWatermark' => 'Watermark',
```

and also

```php
'HELP_CBWatermark' => 'This workflow task will apply a watermark to the image contained in the given field',
```

 (or something like that). This one is used in the Help screen.